src/probe/probe.c

```diff
@@ -1021,10 +1066,12 @@ int probe_point_add(uint32_t count, struct probe_point *probe)
 
 				return -EBUSY;
 			}
-		} else if (probe[i].purpose == PROBE_PURPOSE_EXTRACTION) {
+		} else if (probe[i].purpose == PROBE_PURPOSE_EXTRACTION ||
+			   probe[i].purpose == PROBE_PURPOSE_LOGGING) {
```

Comment:
Earlier the code have already checked the validity of the purpose (INJECTION, EXTRACTION or LOGGING). Here if the purpose was not INJECTION then it is either EXTRACTION or LOGGING, thus a simple `} else {` is fine?

Reply:
Applies to the original code as well, not related to this patch.

Reply:
Will fix.